package.json

```diff
@@ -56,11 +56,12 @@
     "node-sass": "^4.12.0"
   },
   "devDependencies": {
+    "acorn": "^7.0.0",
```

Comment:
@phillipluther When I updated eslint, I noticed the `acorn` was an unresolved peer dependency for one of it's dependencies. I am not sure on the proper procedure here. Does this mean we should include it in out package, or is that a responsibility of the consumer?

Reply:
That's super weird; I pulled down the branch, did a rimraf on `node_modules` and reinstalled and didn't see this warning. Can you confirm it's still showing with a clean install?

If it's not just a matter of NPM getting confused after the ESLint upgrade, I vote we include it. That pains me, because we're not explicitly using it ... but it's a dev dep so it wouldn't add bloat to the dist package and would only affect people who clone/fork the repo.

Reply:
I did the same thing you did and didn't get the warning either. Not sure why I got it before. Also, the checks passed in the pipeline without it so, taking it out :). Also have you noticed how your approval didn't go away when I pushed up a new commit. I wish gitlab did that.